airflow/api/client/json_client.py

```diff
@@ -24,9 +24,24 @@
 
 
 class Client(api_client.Client):
-    """Json API client implementation."""
+    """Json API client implementation.
 
-    def _request(self, url, method="GET", json=None):
+    This client is used to interact with a Json API server and perform various actions
+    such as triggering DAG runs,deleting DAGs, interacting with pools, and getting lineage information.
+
+    :param api_base_url: The base URL for the Json API server.
+    :param session: A session object to use for making HTTP requests.
```

Comment:
Is the `api_base_url` and `session` necessary here?

Reply:
I guess so, because it is inherited from the Base one

Reply:
I believe the one we have in the base is enough

Reply:
Alright, let's drop it

Reply:
Done